### ISSUE TITLE
Add a button to resume reading after focusing navigation menu

### ISSUE THREAD

**Issue by nvdaes** [edrlab/thorium-reader#1010]
It's possible to explore the bookmarks or table of contents and then the user may want to resume reading from the last position. A button with that position could be added to the navigation menu with this purpose, and also it maybe saved after closing the reader.

**Comment by nvdaes** [comment]
This can be achieved in NVDA disabling the option of following the system cursor and then a command to route the system cursor to the review cursor. If this issue cannot be fixed, at least creating a wiki article for screen readers maybe useful.

**Comment by danielweck** [comment]
This feature already exists, it is the "underscore underscore" special hidden link the displayed document, which always references the last recorded reading location. The shortcut `CTRL` + `F10` positions the keyboard focus immediately before the main content area, and a single `tab` key stroke (or other screen reader access mode) reaches the "underscore underscore" link which when followed will jump to the last reading location (i.e. any position previously navigated via the Tablet Of Contents, Bookmarks, Landmarks, Page Lists, or simply "turning pages" in a reflowable or fixed layout publication, but also mouse clicking inside the document text which sets the reading location at the pointed paragraph / phrase, as well as creating a document selection range with the mouse or keyboard).
